### DISCUSSION TITLE
#48 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Emilyxiannv** [tog-ETH-er/togETHer#29] [General]
Agenda:

Community updates
Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by 0x-stan** [comment]
# Dapp-Learning

## Gitcoin15

https://gitcoin.co/grants/3414/dapp-learning-developer-group

## repo update

8个PR #831 ～ #838

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《EthStorage Scaling Ethereum Storage》 by qizhou
  - https://www.youtube.com/watch?v=vR1JYfbU0QQ
  - https://www.youtube.com/watch?v=-ueouNhw0-s

- 《EulerFinance 代码解析》by 三火
  - https://www.youtube.com/watch?v=t4PwA4iFVVM
  - https://www.youtube.com/watch?v=9CmzeIbKM2Y

## MIT 公开课翻译

- MIT class 23 Digital ID
  - https://mp.weixin.qq.com/s/1EmHO7pNau7cS_ZHxKQHew

**Comment by luckyyang** [comment]
### Rebase
- web3 daily 更新 14 期 https://rebase.network/posts/category/web3daily
- 招聘: 4 个公司发布多个职位 https://github.com/rebase-network/who-is-hiring/issues
- 活动: 支持 3 个活动 https://github.com/rebase-network/event-pool/issues

**Comment by Thegaram** [comment]
### Crypto Tech Night

- 8th September: CTN#17 Paradigm CTF with Comcat (DappLearning), 0xaa (WTF Academy), Solidity Go (WTF Academy), and Neek (Amber Group) https://www.bilibili.com/video/BV1RD4y1v7TL

![image](https://user-images.githubusercontent.com/7571518/191013231-84a24150-98c4-45c2-9e53-e4d7b35130f0.png)

- 15th September: Assist ECN with "The Merge" event.

- Gitcoin grant application: https://gitcoin.co/grants/7331/crypto-tech-night-blockchain-developer-enthusiast

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网内容 review 进行中

**Comment by 0x68616f79756e** [comment]
ETHPlanet
支持 ETH Tech Weeks 活动和课程。

**Comment by Emilyxiannv** [comment]
### DAOSquare
PHoenix 产品内测中，将于9月30日上线V1版本；
Phoenix 是一个 DAOSquare 社区所拥有的孵化器。它由 VentureDAO、社区身份、孵化服务市场、社区福利等模块构成。任何初创项目都可以向 Phoenix 寻求帮助，任何人也都可以成为社区成员参与投资和孵化工作。所有参与者都可以通过社区机制拥有和享受Phoenix 所创造的价值。

**Comment by freeyao** [comment]
原语里弄

报告
- 以太坊协议层的七年之变
[链接](https://mirror.xyz/0xbeC73ba0817403cd11C11bE891D671EA30443562/aAjZCMWwd_GEfqiLJNVzqvpOTuk0202Iotyx-KQ-oHI)
- TSS 开放知识库共建
[链接]( https://shoutout.wix.com/so/32OC8VFhC?languageTag=zh)
